packages/@aws-cdk/aws-eks/README.md

```diff
@@ -253,44 +254,109 @@ cluster.addFargateProfile('MyProfile', {
 });
 ```
 
-To create an EKS cluster that **only** uses Fargate capacity, you can use
-`FargateCluster`.
+> For a complete API reference visit [`FargateProfileOptions`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-eks.FargateProfileOptions.html)
+
+You can also directly use the `FargateProfile` construct to create profiles under different scopes:
+
+```ts
+new eks.FargateProfile(scope, 'MyProfile', {
+  cluster,
+  ...
+});
+```
+
+> For a complete API reference visit [`FargateProfileProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-eks.FargateProfileProps.html)
 
-The following code defines an Amazon EKS cluster without EC2 capacity and a default
-Fargate Profile that matches all pods from the "kube-system" and "default" namespaces. It is also configured to [run CoreDNS on Fargate](https://docs.aws.amazon.com/eks/latest/userguide/fargate-getting-started.html#fargate-gs-coredns) through the `coreDnsComputeType` cluster option.
+To create an EKS cluster that **only** uses Fargate capacity, you can use `FargateCluster`.
+The following code defines an Amazon EKS cluster with a default Fargate Profile that matches all pods from the "kube-system" and "default" namespaces. It is also configured to [run CoreDNS on Fargate](https://docs.aws.amazon.com/eks/latest/userguide/fargate-getting-started.html#fargate-gs-coredns).
 
 ```ts
 const cluster = new eks.FargateCluster(this, 'MyCluster', {
   version: eks.KubernetesVersion.V1_16,
 });
-
- // apply k8s resources on this cluster
-cluster.addManifest(...);
 ```
 
 **NOTE**: Classic Load Balancers and Network Load Balancers are not supported on
 pods running on Fargate. For ingress, we recommend that you use the [ALB Ingress
 Controller](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html)
 on Amazon EKS (minimum version v1.1.4).
 
-### Spot Capacity
+#### 3) Self Managed Auto Scaling Groups
+
+Another way of allocating capacity to an EKS cluster is by using self-managed Auto Scaling Groups.
+EC2 instances that are part of the auto-scaling group will serve as worker nodes for the cluster.
+This type of capacity is also commonly referred to as *EC2 Capacity** or *EC2 Nodes*.
+
+For a detailed overview please visit [Self Managed Nodes](https://docs.aws.amazon.com/eks/latest/userguide/worker.html).
+
+These self-managed auto-scaling groups provide some additional capabilities over the managed node group, as detailed below.
+However, as they incur an extra maintenance overhead, they are usually not considered a best practice.
```

Comment:
This is a new statement I added. Want to make sure we comfortable saying this.

Reply:
I'd just take the wording from the AWS docs

Reply:
EKS docs don't convey this messaging. I'll just remove it.